source/systemUtils.py

```diff
@@ -63,6 +63,45 @@ def hasUiAccess():
 		ctypes.windll.kernel32.CloseHandle(token)
 
 
+TokenOrigin = 17
+
+
+class TOKEN_ORIGIN(ctypes.Structure):
+	_fields_ = [("OriginatingLogonSession", ctypes.c_ulonglong)]
+
+
+def getProcessTokenOrigin(processHandle: int) -> TOKEN_ORIGIN:
```

Comment:
A docstring describing what this is getting, and what it is used for is needed here.

Reply:
I decided to merge the approach to get the logon session ID in one function, it looks more obvious now.